src/styles/header.scss

```diff
@@ -0,0 +1,76 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: Roboto, serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+
+  .logo-container {
+    width: 40px;
+    height: 40px;
+
+    .logo {
+      display: block;
+      width: 100%;
+      height: auto;
+    }
+  }
+
+  .nav {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+
+    transition: color 0.3s ease;
+
+    .nav__item {
+      margin-right: 20px;
+      height: 60px;
+      line-height: 60px;
+      padding: 0;
+
+      .nav__link {
+        text-decoration: none;
+        color: $nav-link-color;
+        height: 60px;
+
+        &:hover {
+          color: $blue-color;
+        }
+      }
+    }
+
+    .is-active {
+      color: $blue-color;
+
+      &::after {
+        content: '';
+        height: 4px;
+        display: block;
+        background-color: $blue-color;
```

Comment:
The variables `$nav-link-color` and `$blue-color` are used but not defined in this file. Ensure these variables are defined in a variables file or imported correctly to avoid compilation errors.